configuration-reference/functions/firstwithtime.md

```diff
@@ -0,0 +1,24 @@
+---
+description: This section contains reference documentation for the firstwithtime function.
+---
+
+# firstwithtime
+
+Get the first value of `dataColumn` where the `timeColumn` is used to define the time of `dataColumn` and the `dataType` specifies the type of `dataColumn`, which can be `BOOLEAN`, `INT`, `LONG`, `FLOAT`, `DOUBLE`, `STRING`
```

Comment:
I feel it is a little bit more clear this way: Get the value of `dateColumn` with the smallest `timeColumn` value...